tests/test_cli.py

```diff
@@ -102,14 +108,26 @@ def setUpClass(cls):
             'en.yaml': '---\n'
                        'a: true\n'
                        'A: true'
-        })
+        }
+
+        if utf8_paths_supported():
+            # non-ASCII chars
+            workspace_def['non-ascii/éçäγλνπ¥/utf-8'] = (
```

Comment:
I still kept this part in, because this way we only need to disable two out of ~25 tests